src/main/java/org/zalando/nakadi/security/ScopeHelper.java

```diff
@@ -0,0 +1,22 @@
+package org.zalando.nakadi.security;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.zalando.nakadi.exceptions.IllegalScopeException;
+
+import java.util.Optional;
+import java.util.Set;
+
+public class ScopeHelper {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ScopeHelper.class);
+
+    public static void checkScopes(final Optional<Set<String>> scope, final Client client) {
```

Comment:
scope because it is optional (one) and also I think it is more readable on the next line. i've added unchecked exception to add for processing in ExceptionHandling.java